minstack.java

```diff
@@ -0,0 +1,35 @@
+// Problem 2:
+//Design MinStack (https://leetcode.com/problems/min-stack/)
+
+// Time Complexity : O(1)
```

Comment:
Kindly mention the complexity of individual functions. the way it is discussed in the class